lighthouse-core/computed/metrics/lantern-largest-contentful-paint.js

```diff
@@ -65,6 +74,20 @@ class LanternLargestContentfulPaint extends LanternMetric {
     );
   }
 
+  /**
+   * @param {LH.Gatherer.Simulation.Result} simulationResult
+   * @return {LH.Gatherer.Simulation.Result}
+   */
+  static getEstimateFromSimulation(simulationResult) {
+    const nodeTimesNotOffscreenImages = Array.from(simulationResult.nodeTimings.entries())
+      .filter(entry => !LanternLargestContentfulPaint.isLowPriorityImageNode(entry[0]));
```

Comment:
what do you think about a
```suggestion
    const endTimesNotOffscreenImages = Array.from(simulationResult.nodeTimings.entries())
      .filter(entry => !LanternLargestContentfulPaint.isLowPriorityImageNode(entry[0]))
      .map(entry => entry[1].endTime);
```

Reply:
sure, done